security/pkg/nodeagent/caclient/providers/citadel/testdata/notexpired-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This will expire in 1 month, causing all unit tests to fail. Please make it 10+ years, and add a comment indicating the command used to generate it

Reply:
ok I will change it